src/components/container/editContainerNote.jsx

```diff
@@ -40,9 +40,10 @@ export class EditContainerNote extends Component {
   
     render () {
         return (
-                <TextField
+            <TextField
                 multiline margin="none" placeholder="Add Note here"
                 autoFocus={true}
+                onBlur={() => this.props.handleNoteEnter(null)}
```

Comment:
is there a reason why you are passing null here?

Reply:
Addressed your comment. Thanks!

Reply:
nice.